### ISSUE TITLE
TaskTree raise an exception when called

### ISSUE THREAD

**Issue by nicolasfara** [dorongold/gradle-task-tree#49]
Hi,
first of all thanks for this plugin: it's very convenient!

I'm currently using Gradle 7.6 with this plugin.
When launching the task `./gradlew build taskTree`, the following exception is raised:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':taskTree'.
> Cannot cast object 'org.gradle.execution.plan.DefaultFinalizedExecutionPlan@e45462c' with class 'org.gradle.execution.plan.DefaultFinalizedExecutionPlan' to class 'org.gradle.execution.plan.DefaultExecutionPlan'
```

This is the repo in which the plugin fails: https://github.com/nicolasfara/pulverization-framework

Is this maybe related to a new version of Gradle that is not already supported by the plugin?

**Comment by roshea1-chwy** [comment]
Looks like https://gitlab.com/barfuin/gradle-taskinfo is experiencing the same issue on Gradle 7.6 https://gitlab.com/barfuin/gradle-taskinfo/-/issues/20#note_1213772592

**Comment by dorongold** [comment]
Fixed with plugin version 2.1.1.
